src/lib/processor.py

```diff
@@ -1,7 +1,10 @@
 import do
 import actions
 from exceptions import *
```

Comment:
Hmm, in my testing it would always import the builtin module exceptions instead of yours. Wonder why you see a difference.